src/lib/components/Swap/Summary/index.tsx

```diff
@@ -70,21 +72,27 @@ const Body = styled(Column)<{ open: boolean }>`
   }
 `
 
+const updateMessage = { message: <Trans>Price updated</Trans>, action: <Trans>Accept</Trans> }
```

Comment:
nit: name this `update` or `priceUpdate` or `tradeUpdate`

Reply:
i get confused whenever i see `update`, so i like priceUpdate